runtime/src/main/scala/akka/grpc/javadsl/GrpcErrorResponse.scala

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (C) 2018-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.grpc.javadsl
+
+import java.lang.{ Iterable => jIterable }
```

Comment:
while there indeed also exists an `Iterable` in Scala, I think that's used rarely enough that you can just use `java.lang.Iterable` without an alias like this

Reply:
I like using the j-prefix when referencing Java types in a Scala source file just to indicate that something slightly unusual is going on.

That said, I've removed the prefix everywhere except in GrpcExceptionHelper, where there is enough Java/Scala conversion code that it's nice to have consistent prefixes on the parameter and return types.